### DISCUSSION TITLE
Any way to disable relay that does not include unenrollment?

### DISCUSSION THREAD

**Discussion by ScopfyScythe** [3kh0/ext-remover#373] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Any way to disable relay that does not include unenrollment?

**Comment by Huntlb** [comment, marked as answer]
how about reading the main page of this

**Reply by 3kh0** [reply]
lol

**Comment by Alex2Mayhem** [comment]
Here is a way.

Step 1. Don't unenroll.
Step 2. Don't unenroll.
Step 3. Still don't even try to unenroll.
Step 4. Wait, and don't unenroll.

**Reply by Aka-but-nice** [reply]
Wow, it worked thanks dude.
